requirements.txt

```diff
@@ -0,0 +1,9 @@
+# Runtime requirements for qibo
```

Comment:
I thought the plan was to remove this file and have the dependencies in `setup.py` directly. Do you prefer the current approach that has requirements.txt? I am fine with either way.

Reply:
Yes, but then I realized that the test for wheels was not working as expected. So to avoid code duplication I have restore the previous mechanism.